src/hazelcore/extractions/ocaml/OCamlExtraction_Pat.re

```diff
@@ -0,0 +1,98 @@
+type t = string;
```

Comment:
some of the same concerns raised above with expressions are relevant here, won't repeat